src/Compiler/Driver/fsc.fs

```diff
@@ -569,6 +569,7 @@ let main1
             exiter.Exit 1
 
     if tcConfig.showTimes then
+        Caches.CacheMetrics.CaptureStatsAndWriteToConsole() |> disposables.Register
```

Comment:
I would like us to have a think if we can decouple the `Cache` from the `Meters`.
Just like we have the `Activity` module and then any code path is "invited" to use, maybe `Meters` could have the same treatment within the compiler.

(I know it uses the underlying .NET primitive, but it will then be more inviting for use at other contributions. e.g. IO access seems like a natural candidate to use meters, too)